### ISSUE TITLE
Syndactyly, Type I (MONDO:0008512) incorrect exactmatch OMIM ID and ORPHA ID

### ISSUE THREAD

**Issue by lcdaugherty** [monarch-initiative/mondo#1956]
Possible error in mapping?

Syndactyly, Type I (MONDO:0008512) currently has 
exactmatch http://identifiers.org/omim/185900 and http://purl.obolibrary.org/obo/Orphanet_93402

The Orphanet term (ORPHA:93402) represents the parent term, Syndactyly type 1 or Zygodactyly – whereas the OMIM term (OMIM:185900) represents a daughter term, zygodactyly type 3 (ORPHA:295191).

Suggest OMIM:185900 should map to zygodactyly type 3 (MONDO:0017544) and not Syndactyly, Type I (MONDO:0008512).

**Comment by nicolevasilevsky** [comment]
Hi @lcdaugherty 
Where do you see that OMIM:185900 is mapped to zygodactyly type 3 (ORPHA:295191)?

On the OMIM [page](https://www.omim.org/clinicalSynopsis/185900), I see link outs to [Craniosynostosis, Philadelphia type](https://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=EN&Expert=1527) and [Syndactyly type 1](https://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=EN&Expert=93402) in Orphanet.

**Comment by nicolevasilevsky** [comment]
pinging @lcdaugherty

**Comment by lcdaugherty** [comment]
@nicolevasilevsky Sorry, for delay responding to this, it looks like my notification settings were not set up right- I had changed them (muted) when I moved to my old job but now my new job requires me to look at ontologies again :) I have now updated my settings, so I should be a bit more responsive in future!
Looking at this again, I thought OMIM:185900 should map to MONDO:0017544 and not MONDO:0008512.
the reason I was querying it is that MeSH C566096 https://www.ncbi.nlm.nih.gov/mesh/?term=C566096 used to be mapped MONDO:0008512, but has since been removed so I was trying to figure out why.
